### PR TITLE
Refactor/airdrop

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206250953909089

### CODE REVIEW

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/airdrop/airdrop-form/airdrop-preview-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      <Box
+        maxWidth="95%"
+        borderRadius="xs"
+        width="26.875rem"
+        minHeight="46rem"
+        alignItems="center"
+        display="inline-flex"
+        justifyContent="space-between"
+        flexDirection="column"
+        boxShadow="dropShadow.2xl"
```
Comment (KipandaJr) [inline views/airdrop/airdrop-form/airdrop-preview-modal.tsx:40]: u need this ?

File: views/airdrop/airdrop-form/airdrop-preview-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              p="0.75rem"
+              borderRadius="xs"
+              display="flex"
+              alignItems="center"
+              justifyContent="space-between"
+            >
+              <Box display="flex" alignItems="center" gap="m">
+                <Box
+                  bg="black"
+                  color="white"
```
Comment (KipandaJr) [inline views/airdrop/airdrop-form/airdrop-preview-modal.tsx:84]: check this colors

File: views/airdrop/airdrop-form/airdrop-summary.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            color="#000000A3"
+          >
+            Delivery Method
+          </Typography>
+          <Box display="flex" justifyContent="center" alignItems="center">
+            <Typography
+              variant="body"
+              size="medium"
+              color="onSurface"
+              mr="0.5rem"
```
Comment (KipandaJr) [inline views/airdrop/airdrop-form/airdrop-summary.tsx:41]: check this...

File: views/components/select-nft-modal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            label="Search NFT"
+            placeholder="NFT Name"
+            {...register('search')}
+            fieldProps={{ height: '3.5rem', mb: 'm', borderRadius: 'xs' }}
+            Prefix={<SearchSVG maxWidth="1rem" maxHeight="1rem" width="100%" />}
+          />
+        </Box>
+      </Box>
+      <Motion
+        bg="#B6C4FF33"
```
Comment (KipandaJr) [inline views/components/select-nft-modal/index.tsx:75]: check this color

File: views/components/select-nft-modal/nft-modal-item.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      isFavorite
+        ? favoriteTokens.filter((favType) => favType !== type)
+        : [...favoriteTokens, type]
+    );
+
+  return (
+    <Box
+      p="xl"
+      display="flex"
+      color="textSoft"
```
Comment (KipandaJr) [inline views/components/select-nft-modal/nft-modal-item.tsx:38]: this color dosen't exist in new theme

File: views/components/select-token-modal/select-token-modal-base.tsx [hunk 6]
```diff
@@ -18,38 +16,40 @@ import {
...
+  <Tag
+    size="small"
+    display="flex"
+    variant="outline"
+    justifyContent="center"
+    onClick={handleSelectToken}
+    PrefixIcon={
+      <Box
+        bg="black"
+        color="white"
```
Comment (KipandaJr) [inline views/components/select-token-modal/select-token-modal-base.tsx:31]: check this colors

File: views/components/select-token-modal/token-modal-item.tsx [hunk 7]
```diff
@@ -34,16 +34,16 @@ const TokenModalItem: FC<TokenModalItemProps> = ({
...
       color="textSoft"
       cursor="pointer"
       alignItems="center"
-      nHover={{ bg: 'rgba(0, 83, 219, 0.08)' }}
       justifyContent="space-between"
-      bg={selected ? 'rgba(0, 83, 219, 0.08)' : 'unset'}
       onClick={selected ? undefined : onClick}
       transition="background 500ms ease-in-out"
+      nHover={{ bg: 'rgba(0, 83, 219, 0.08)' }}
+      bg={selected ? 'rgba(0, 83, 219, 0.08)' : 'unset'}
```
Comment (KipandaJr) [inline views/components/select-token-modal/token-modal-item.tsx:46]: try see if this colors dosent exist on theme

File: views/my-coins/table-wrapper/index.tsx [hunk 8]
```diff
@@ -8,28 +8,26 @@ const TableWrapper: FC<PropsWithChildren<TableWrapperProps>> = ({
...
-      mb="xs"
-      borderRadius="m"
-      mx={[0, '11xl']}
-      overflow="hidden"
-      bg="lowestContainer"
-      width={['calc(100vw - 3rem)', 'auto']}
-      boxShadow="0px 24px 46px -10px rgba(13, 16, 23, 0.16)"
+      p="1.5rem"
+      fontWeight="500"
+      fontSize={['1.375rem', '1.375rem', '1.375rem', '2rem']}
```
Comment (KipandaJr) [inline views/my-coins/table-wrapper/index.tsx:24]: `font-size` on theme

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: views/components/select-token-modal/token-modal-item.tsx (hunk 7 above)
Comment (git-marcopitra) [reply views/components/select-token-modal/token-modal-item.tsx:46]: there is not alpha colors

File: views/components/select-nft-modal/index.tsx (hunk 4 above)
Comment (git-marcopitra) [reply views/components/select-nft-modal/index.tsx:75]: There is not alpha color

**Review by MarioBatalha — APPROVED** [review]

**Pull request merged**